skimage/color/colorconv.py

```diff
@@ -1742,7 +1742,7 @@ def lch2lab(lch, *, channel_axis=-1):
     Examples
     --------
     >>> from skimage import data
-    >>> from skimage.color import rgb2lab, lch2lab
+    >>> from skimage.color import rgb2lab, lch2lab, lab2lch
     >>> img = data.astronaut()
     >>> img_lab = rgb2lab(img)
     >>> img_lch = lab2lch(img_lab)
```

Comment:
Since #5505 by @grlee77 restored doctesting on CI, how was this running so far?

Reply:
The doctests will [know about any other functions defined in the same module](https://docs.python.org/3/library/doctest.html#what-s-the-execution-context). So strictly speaking we don't have to import `rgb2lab`, `lch2lab` or `lab2lch` here, but I think it is still best to do so as it is helpful to users trying to copy from these examples.